modules/hcp-hc-objects.adoc

```diff
@@ -188,7 +188,7 @@ status:
 <2> Replace `<hosted_cluster_namespace>` with the name of your hosted cluster namespace.
 <3> The `imageContentSources` section contains mirror references for user workloads within the hosted cluster.
 <4> Replace `<dns.base.domain.name>` with the DNS base domain name.
-<5> Replace `4.x.y` with the supported {product-title} version you want to use.
+<5> Replace `<4.x.y>` with the supported {product-title} version you want to use.
```

Comment:
On line 157, be sure to put angle brackets around `4.x.y` to match the text in the annotation.